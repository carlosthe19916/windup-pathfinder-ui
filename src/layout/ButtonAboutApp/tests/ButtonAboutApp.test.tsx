import React from "react";
import { shallow } from "enzyme";
import { ButtonAboutApp } from "../ButtonAboutApp";

it("Test snapshot", () => {
  const wrapper = shallow(<ButtonAboutApp />);
  expect(wrapper).toMatchSnapshot();
});
