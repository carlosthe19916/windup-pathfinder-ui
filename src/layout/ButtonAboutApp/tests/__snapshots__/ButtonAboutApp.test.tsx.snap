// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
<Fragment>
  <Button
    aria-label="About button"
    id="aboutButton"
    onClick={[Function]}
    variant="plain"
  >
    <HelpIcon
      color="currentColor"
      noVerticalAlign={false}
      size="sm"
    />
  </Button>
  <AboutModal
    appendTo={null}
    backgroundImageSrc=""
    brandImageAlt="Logo"
    brandImageSrc="logo-navbar.svg"
    className=""
    isOpen={false}
    noAboutModalBoxContentContainer={false}
    onClose={[Function]}
    productName=""
    trademark="COPYRIGHT Â© 2020."
  >
    <TextContent>
      <TextList
        component="dl"
      >
        <TextListItem
          component="dt"
        >
          Source code
        </TextListItem>
        <TextListItem
          component="dd"
        >
          <a
            href="https://github.com/carlosthe19916/windup-pathfinder"
          >
            Github
          </a>
        </TextListItem>
      </TextList>
    </TextContent>
  </AboutModal>
</Fragment>
`;
